src/results-output/results-doc.ts

```diff
@@ -303,6 +303,14 @@ export function append(text: string, onAppended?: OnAppendedCallback): void {
     };
 }
 
+export function clearEditQueue(): void {
+    editQueue = [];
+}
+
+export function getEditQueueLength(): number {
+    return editQueue.length;
+}
+
```

Comment:
An exported function names something like `stopPrinting()` makes more sense to me. Or maybe that makes you think there should be a `resumePrinting()` as well... Hmmm maybe `discardPendingPrints()` and that function would take care of the business.

Reply:
Oh I see what you mean now. I'll make changes.